Application/EdFi.Ods.WebApi/appsettings.json

```diff
@@ -28,6 +28,7 @@
         "OdsTokens": "",
         "OdsDatabaseTemplateName": "minimal",
         "PopulatedTemplateDBTimeOutInSeconds": "600",
+        "LoadProfilesFromDatabase": true,
```

Comment:
keep the logic for both reading from the database and profiles found in assemblies for now